.github/workflows/chart-release.yaml

```diff
@@ -7,8 +7,13 @@ on:
       branches:
```

Comment:
Thanks for this PR @naveensrinivasan . In fact, this file can be removed from the repo (will do in a follow-up PR) - the Helm chart is now moved to a separate repo: https://github.com/localstack/helm-charts